sdl-core/src/main/scala/io/smartdatalake/util/hdfs/Partition.scala

```diff
@@ -209,6 +209,9 @@ object PartitionLayout {
   }
 
   def extractPartitionValues(partitionLayout: String, path: String): PartitionValues = {
+    val cleanPath = if (Environment.caseSensitive) path
+    else "^(.*?)=".r.replaceAllIn(path, m => m.group(0).toLowerCase)
```

Comment:
This only matches the beginning of the path, why? Shouldn't we make the whole path lowercase?

Reply:
You mean also the part after = ? I think that would mess up the data. Or are there also possibilities to have a path like partition_X/partition_Y=anything?

Reply:
A path can have multiple partition columns, and i think its not sure that it is relative. Examples would be:
- partition_X=aBC/partition_Y=Def/test.json
- /some/root/path/partition_X=aBC/partition_Y=Def/test.json
I suggest to adapt the Regex as follows:
```suggestion
    else "[\^/]([^/=]*)=".r.replaceAllIn(path, m => m.group(0).toLowerCase)
```